src/search-modal/SearchModal.test.jsx

```diff
@@ -59,7 +64,7 @@ describe('<SearchModal />', () => {
       apiKey: 'test-api-key',
     });
     const { findByText } = render(<RootWrapper />);
-    expect(await findByText('Enter a keyword or select a filter to begin searching.')).toBeInTheDocument();
+    expect(await findByText('Start searching to find content')).toBeInTheDocument();
```

Comment:
Can you add more tests to cover the added functionality for this SearchModal?

Reply:
More tests: 879a3952f2d33caf3fe6b095836e0a1f6bd3a645, 66612bcc7df0638117372aa33b23f382e5fc674a, 58a5c8c6cb456d9ebf7609c2836fd57b690afe3e